File: sample/sample.go
package main

import (
	"fmt"
	"io/ioutil"
	"net/http"
)

type Test struct {
	Some string `json:"some"`
	Yaml int64  `yaml:"yaml"`
}

const (
	number  = 99
	boolean = true
	str     = "some string"
)

func main() {

	if str == "test string" {
		panic(123)
	} else {
		for _, i := range []string{"some"} {
			fmt.Println(i)
		}
	}

	url := "https://httpbin.org/get"
	req, _ := http.NewRequest("GET", url, nil)
	res, err := http.DefaultClient.Do(req)

	if err != nil {
		panic(err)
	}

	defer res.Body.Close()
	body, _ := ioutil.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}

File: sample/sample.ts
var http = require('https');

var options = {
	method: 'GET',
	hostname: 'httpbin.org',
	port: null,
	path: '/get',
	headers: {
		'content-length': '0',
	},
};

export interface Test {
  some: string
  boolean: boolean
}

var req = http.request(options, function (res) {
  var chunks: Array<any> = [];
  let bool = true || false
  let number = 99

	res.on('data', function (chunk: any) {
		chunks.push(chunk);
	});

	if (chunks.length > 0) {
		console.log(chunks);
	} else {
		throw Error('some error');
	}

	res.on('end', function () {
		var body = Buffer.concat(chunks);
		console.log(body.toString());
	});
});

req.end();
